### PR TITLE
Added a Classic Modern format

### PR DESCRIPTION
**Pull request by marthinwurer**
See title. Classic Modern is pre-WAR, so none of the busted walkers, Modern Horizons mistakes, and funky new companions. 

[Discord community](https://discord.gg/q5KgvZ)

### CODE REVIEW

File: Mage.Server.Plugins/Mage.Deck.Constructed/src/mage/deck/ClassicModern.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+import mage.cards.ExpansionSet;
+import mage.cards.Sets;
+import mage.cards.decks.Constructed;
+import mage.sets.EighthEdition;
+import mage.sets.WarOfTheSpark;
+
+import java.util.Date;
+
+/**
+ * Deck validation for modern before WAR was released.
```
Comment (JayDi85) [inline Mage.Server.Plugins/Mage.Deck.Constructed/src/mage/deck/ClassicModern.java:12]: Add web page/wiki url with format description here

File: Mage.Server/release/config/config.xml [hunk 2]
```diff
@@ -149,6 +149,7 @@
         <deckType name="Constructed - Pioneer" jar="mage-deck-constructed.jar" className="mage.deck.Pioneer"/>
         <deckType name="Constructed - Modern" jar="mage-deck-constructed-${project.version}.jar" className="mage.deck.Modern"/>
         <deckType name="Constructed - Modern - No Banned List" jar="mage-deck-constructed-${project.version}.jar" className="mage.deck.ModernNoBannedList"/>
+        <deckType name="Constructed - Classic Modern" jar="mage-deck-constructed.jar" className="mage.deck.ClassicModern"/>
```
Comment (JayDi85) [inline Mage.Server/release/config/config.xml:152]: Missing `-${project.version}`.

File: Mage.Server/release/config/config.xml [hunk 3]
```diff
@@ -149,6 +149,7 @@
         <deckType name="Constructed - Pioneer" jar="mage-deck-constructed.jar" className="mage.deck.Pioneer"/>
```
Comment (JayDi85) [inline Mage.Server/release/config/config.xml:149]: You can fix that (Constructed - Pioneer) too with missing `-${project.version}`.

**Comment by xenohedron** [comment]
Is there still interest in this? I couldn't find any related docs on the web with a quick look.

**Pull request closed without merging**